src/test/java/com/checkout/apm/baloto/BalotoPaymentsTestIT.java

```diff
@@ -33,7 +33,7 @@ void shouldSucceedBalotoPayment() {
         final GetPaymentResponse response = blocking(defaultApi.paymentsClient().getPayment(paymentId));
 
         assertNotNull(response);
-        assertEquals(PaymentStatus.CAPTURED, response.getStatus());
+        //assertEquals(PaymentStatus.CAPTURED, response.getStatus());
```

Comment:
This might work if you get the payment using `protected <T> T blocking(final CompletableFuture<T> future, final Matcher<T> matcher) {` until it the payment returned is CAPTURED.